README.md

```diff
@@ -267,11 +267,9 @@ This is implemented in a way that is reasonably agnostic to the exact GPU-Tensor
 ## WebGL
 
 1. Download the [test scenes](http://dl.fbaipublicfiles.com/habitat/habitat-test-scenes.zip) and extract locally to habitat-sim creating habitat-sim/data.
-1. Download and install [emscripten](https://emscripten.org/docs/getting_started/downloads.html) (version 1.38.42 is verified to work)
-1. Set EMSCRIPTEN in your environment
-   ```bash
-   export EMSCRIPTEN=/pathto/emsdk/fastcomp/emscripten
-1. Build using `./build_js.sh`
+1. Download and install [emscripten](https://emscripten.org/docs/getting_started/downloads.html) (version 2.0.6 is verified to work)
```

Comment:
```suggestion
1. Download and install [emscripten](https://emscripten.org/docs/getting_started/downloads.html) (you need at least version 1.38.42, newer versions such as 2.0.6 work too)
```

1.38.42 is still used on the CIs, no?

I personally did not yet verify that all of magnum works with 2.0+, so [again](https://github.com/facebookresearch/habitat-sim/pull/843) I'm advising to not go too high unless we need to :) The changed text also looked like 1.38.42 doesn't work anymore, which isn't the case.